src/index.html

```diff
@@ -8,106 +8,86 @@
     <meta name="description" content="Essential JS 2" />
     <meta name="author" content="Syncfusion" />
     <link rel="shortcut icon" href="resources/favicon.ico" />
-    <link href="https://fonts.googleapis.com/css?family=Roboto:100,100i,300,300i,400,400i,500,500i,700,700i,900,900i"
-        rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:100,100i,300,300i,400,400i,500,500i,700,700i,900,900i" rel="stylesheet">
     <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet" />
     <!--style reference from node_modules/@syncfusion/ej2/-->
-    <link href="/styles/styles.css" rel="stylesheet" />
-
-    <script src="node_modules/systemjs/dist/system.src.js" type="text/javascript"></script>
+    <link href="./styles/styles.css" rel="stylesheet" />
+    <script src="/node_modules/systemjs/dist/system.src.js" type="text/javascript"></script>
     <script src="system.config.js" type="text/javascript"></script>
-
+    <link rel="stylesheet" href="/node_modules/@syncfusion/ej2-navigations/styles/material.css">
+    <link rel="stylesheet" href="/node_modules/@syncfusion/ej2-layouts/styles/material.css">
 </head>
-
 <body style=" font-family: 'Roboto', sans-serif;">
     <div id="container" style=" background: #FAFAFA;">
-        <div class="row" id="overall-title" style="cursor: default">
-            <p class="page-title">Health <b>Stat</b></p>
-            <img src="../../icons/Healthstat.png" style="padding-left: 1.5%; margin-top: 10px; float:left;">
-            <img src="../../icons/Profile_.png" style="padding-right: 1.5%; margin-top: 10px; float:right">
-            <p id="sub-title" style=" float: right">Hi, John Smith</p>
+        <!-- Appbar area -->
+        <div id="appbar">
+            <div class="appbar-content"><span class="e-avatar custom-avatar e-avatar-circle"><img src="../../icons/Healthstat.png"></span><p class="appbar-title"> Health <b>Stat</b> </p>
```

Comment:
Check whether appbar-content is necessary

Reply:
unnecessary class name appbar has been removed